engo_glfw.go

```diff
@@ -1,4 +1,4 @@
-//+build !netgo,!android
+// +build darwin,!arm,!arm64 linux windows
```

Comment:
I'll take a look into it. I'm not sure you'd need a build tag for the emulator. Also why would I need to exclude android. In this line It's including rather than excluding so I'm not sure I'd need to include `!android`. Thanks for the comment though, appreciate it.